update comparisons in FAQ (#4307)
cdr/code-server: docs/FAQ.md

Theia doesn't allow you to reuse your existing VS Code config.

## What's the difference between code-server and OpenVSCode-Server?

code-server and OpenVSCode-Servr both allow you to access VS Code via a
browser. The two projects also use their own [forks of VS Code](https://github.com/cdr/vscode) to

> leverage modern VS Code APIs and stay up to date with the upsteam version.

Author: @TeffenEllis - you have much more context here, and it very is possible I am not understanding this correctly. would love your 👀

Reviewer: This looks good to me (in general!)

Theia doesn't allow you to reuse your existing VS Code config.

## What's the difference between code-server and OpenVSCode-Server?

code-server and OpenVSCode-Server both allow you to access VS Code via a
browser. The two projects also use their own [forks of VS Code](https://github.com/cdr/vscode) to
leverage modern VS Code APIs and stay up to date with the upsteam version.

> However, OpenVSCode-Server is scoped at only making VS Code avalible in the web browser.

Reviewer 2: ```suggestion
However, OpenVSCode-Server is scoped at only making VS Code available in the web browser.
```
Spelling error

code-server includes some other features:

- password auth
- proxy web ports
- certificate support
- plugin API
- settings sync (coming soon)

For more details, see [this discussion post](https://github.com/cdr/code-server/discussions/4267#discussioncomment-1411583).

> ## What's the difference between code-server and VS Code Codespaces?

Reviewer 2: ```suggestion
## What's the difference between code-server and GitHub Codespaces?
```

> Both code-server and VS Code Codespaces allow you to access VS Code via a

Reviewer 2: ```suggestion
Both code-server and GitHub Codespaces allow you to access VS Code via a
```

> browser. VS Code Codespaces, however, is a closed-source, paid service offered by

Reviewer 2: ```suggestion
browser. GitHub Codespaces, however, is a closed-source, paid service offered by
```

> Microsoft. Codespaces are hosted in Microsoft's cloud.

Reviewer 2: ```suggestion
GitHub. Codespaces can only be purchased through GItHub and are hosted on Microsoft's cloud, Azure.
```
